src/node/panner.rs

```diff
@@ -4,34 +4,76 @@ use std::sync::Arc;
 use crate::context::{AudioContextRegistration, AudioParamId, BaseAudioContext};
 use crate::param::{AudioParam, AudioParamOptions};
 use crate::render::{AudioParamValues, AudioProcessor, AudioRenderQuantum};
-use crate::{AtomicF32, SampleRate};
+use crate::{AtomicF64, SampleRate};
 
 use super::{
     AudioNode, ChannelConfig, ChannelConfigOptions, ChannelCountMode, ChannelInterpretation,
 };
 
-/// Options for constructing a PannerNode
+#[derive(Copy, Clone, Debug)]
+pub enum PanningModelType {
```

Comment:
I will spend some precious time tomorrow to check if we now have all enums and options conforming to the spec and we avoid breaking changes in the future 😄 

Probably should mention all unused options in the rustdocs. If we annotate them with `TODO(unimplemented)` or something we can make a nice tracking issue out of them